nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -344,26 +353,45 @@ def apply(
                     and isinstance(reduction_axes, tuple)
                     and len(reduction_axes) != 1
+                    and node.node_name not in ignored_names
                 ):
                     # NNCF supports multiple reduction axes only for ops with group_size != -1.
-                    # Convolution ops are always quantized to 8-bits (without groups).
+                    # Convolution ops are always kept in backup mode.
                     # Embedding layers are quantized to 4-bits only if all_layers=True.
                     # MatMul ops can't have multiple reduction axes.
                     nncf_logger.warning(
                         f"Weight compression expects a single reduction axis, but {len(reduction_axes)} given. "
                         f"Weight shape: {weight_shape}, reduction axes: {reduction_axes}, "
-                        f"node name: {node.node_name}. The node will be asymmetrically quantized to 8 bits."
+                        f"node name: {node.node_name}. The node will be in {self._backup_mode} mode."
                     )
 
+                if node.node_name in ignored_names or self._backup_mode == BackupMode.NONE:
+                    wc_config = None
+                else:
+                    mode = (
+                        CompressWeightsMode.INT8_ASYM
+                        if self._backup_mode == BackupMode.INT8_ASYM
+                        else CompressWeightsMode.INT8_SYM
+                    )
+                    wc_config = WeightCompressionConfig(mode=mode)
                 weight_params = WeightCompressionParameters(
-                    weight_name, node, weight_port_id, weight_size, reduction_axes
+                    weight_name, node, weight_port_id, weight_size, reduction_axes, wc_config
                 )
+                if node.node_name in ignored_names:
+                    ignored_scope_weight_params.append(weight_params)
+                    continue
                 all_weight_params.append(weight_params)
                 weight_names.add(weight_name)
 
         ratio_defining_params = self._get_ratio_defining_params(all_weight_params, is_last_layer_shared)
         self._set_weight_compression_config(ratio_defining_params, model, graph, activations)
-        nncf_logger.info(self._get_bitwidth_distribution_str(all_weight_params, ratio_defining_params))
+        nncf_logger.info(
+            self._get_bitwidth_distribution_str(all_weight_params + ignored_scope_weight_params, ratio_defining_params)
+        )
+        # Filter the weight parameters that should remain in their original floating-point precision
+        all_weight_params = [w_params for w_params in all_weight_params if w_params.compression_config is not None]
+        # Remove nodes in the ignored scope from nodes_to_compress
+        nodes_to_compress = [node for node in nodes_to_compress if node.node_name not in ignored_names]
```

Comment:
Then it makes sense to collect statistics for this limited list of nodes:
```
if dataset is not None and self._sensitivity_metric != SensitivityMetric.WEIGHT_QUANTIZATION_ERROR:
            activations = self._get_activations(dataset, self._subset_size, nodes_to_compress, graph, model)
```

Reply:
To be more consistent, I'd extract the list of nodes to compress from `all_weight_params`. It would guarantee the same content in both lists.